x-pack/plugins/security_solution/public/overview/components/overview_risky_host_links/risky_hosts_enabled_module.tsx

```diff
@@ -9,20 +9,20 @@ import React, { useMemo } from 'react';
 import { RiskyHostsPanelView } from './risky_hosts_panel_view';
 import type { LinkPanelListItem } from '../link_panel';
 import { useRiskyHostsDashboardLinks } from '../../containers/overview_risky_host_links/use_risky_hosts_dashboard_links';
-import type { HostsRiskScore } from '../../../../common/search_strategy';
+import type { HostRiskScore } from '../../../../common/search_strategy';
 
-const getListItemsFromHits = (items: HostsRiskScore[]): LinkPanelListItem[] => {
-  return items.map(({ host, risk_stats: riskStats, risk: copy }) => ({
+const getListItemsFromHits = (items: HostRiskScore[]): LinkPanelListItem[] => {
+  return items.map(({ host }) => ({
```

Comment:
```suggestion
  return items.map(( item ) => ({
```

Reply:
Why we should not spread `item`?

Reply:
mm, we only need this if we are going to apply my suggested change for the next two lines.